person.rb

```diff
@@ -1,6 +1,6 @@
 require './nameable'
 class Person < Nameable

```

Comment:
- [ ] Please add a method to add a rental in this `person` class.
```ruby
def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
end
```
Please note that the `self` keyword is referring to the `person` class